lib/common/buffer-utils.ts

```diff
@@ -19,16 +14,16 @@ const typedArrays = {
   Float64Array
 }
 
-function getType (value) {
+function getType (value: ArrayBuffer | ArrayBufferView) {
```

Comment:
```suggestion
function getType (value: BufferSource) {
```

https://heycam.github.io/webidl/#BufferSource